source/recommended-reading.md

```diff
@@ -35,3 +35,13 @@ A series of posts on basics of how packaging with Nix works.
 ## [awesome-nix](https://nix-community.github.io/awesome-nix/)
 
 A collection of Nix resources.
+
+## Other videos
+
+* [Nix Fundamentals](https://www.youtube.com/watch?v=m4sv2M9jRLg) by Jakub Grzegorz Sokolowski - 05/2020
+* [Nix Shell Overview](https://www.youtube.com/watch?v=SGekN4pDExY) by Wil T - 04/2021
+* [How to update a package](https://www.youtube.com/watch?v=D_IZ2EfW_8U) by Jon Ringer - 11/2020
+* [How to create a patch for any package](https://www.youtube.com/watch?v=5K_2RSjbdXc) by Jon Ringer - 11/2020
+* [Overlays](https://www.youtube.com/watch?v=dGAL3gMXvug) by Jon Ringer - 08/2020
+* [Adding a module to NixOS](https://www.youtube.com/watch?v=bkDYmvKINm8) by Jon Ringer - 01/2022
+* [Upgrade kernel](https://www.youtube.com/watch?v=Zi_vbddNXtg) by Jon Ringer - 09/2020
```

Comment:
Adapted style to the rest of the page, using ISO 8601 dates.

```suggestion
* [Nix Fundamentals](https://www.youtube.com/watch?v=m4sv2M9jRLg) (Jakub Grzegorz Sokolowski, 2020-05)
* [Nix Shell Overview](https://www.youtube.com/watch?v=SGekN4pDExY) (Wil T, 2021-04)
* [How to update a package](https://www.youtube.com/watch?v=D_IZ2EfW_8U) (Jon Ringer, 2020-11)
* [How to create a patch for any package](https://www.youtube.com/watch?v=5K_2RSjbdXc) (Jon Ringer, 2020-11)
* [Overlays](https://www.youtube.com/watch?v=dGAL3gMXvug) (Jon Ringer, 2020-08)
* [Adding a module to NixOS](https://www.youtube.com/watch?v=bkDYmvKINm8) (Jon Ringer, 2022-01)
* [Upgrade kernel](https://www.youtube.com/watch?v=Zi_vbddNXtg) (Jon Ringer, 2020-09)
```

Not sure if the month is even relevant after two years. I don't mind including it, but not having it resolves the date formatting issue. Feel free to choose one.

```suggestion
* [Nix Fundamentals](https://www.youtube.com/watch?v=m4sv2M9jRLg) (Jakub Grzegorz Sokolowski, 2020)
* [Nix Shell Overview](https://www.youtube.com/watch?v=SGekN4pDExY) (Wil T, 2021)
* [How to update a package](https://www.youtube.com/watch?v=D_IZ2EfW_8U) (Jon Ringer, 2020)
* [How to create a patch for any package](https://www.youtube.com/watch?v=5K_2RSjbdXc) (Jon Ringer, 2020)
* [Overlays](https://www.youtube.com/watch?v=dGAL3gMXvug) (Jon Ringer, 2020)
* [Adding a module to NixOS](https://www.youtube.com/watch?v=bkDYmvKINm8) (Jon Ringer, 2022)
* [Upgrade kernel](https://www.youtube.com/watch?v=Zi_vbddNXtg) (Jon Ringer, 2020)
```

Reply:
> Not sure if the month is even relevant after two years

Good point :+1: